src/main/snailtracer.yml

```
@@ -0,0 +1,8 @@
+signextend:
```

```suggestion
snailtracer:
```
The state test format convention is the top level dict has single key of the name of the file.

We should also add a check for the python script (can be separate PR).